src/web/src/views/workspace/WSEditorCommandArgumentsContent.tsx

```diff
@@ -2175,6 +2190,8 @@ function decodeArg(response: any): { arg: CMDArg, clsDefineMap: ClsArgDefinition
         idPart: response.idPart,
         prompt: prompt,
         configurationKey: response.configurationKey,
+        supportEnumExtension: response.enum?.support_extension || response.item?.enum?.support_extension || false,
```

Comment:
I think it should be `supportExtension` in response. Please verify it.
```suggestion
        supportEnumExtension: response.enum?.supportExtension || response.item?.enum?.supportExtension || false,
```